aiida/cmdline/commands/cmd_archive.py

```diff
@@ -170,11 +165,10 @@ def create(
     elif archive_format == 'null':
         export_format = 'null'
 
-    if verbosity in ['DEBUG', 'INFO']:
-        set_progress_bar_tqdm(leave=(verbosity == 'DEBUG'))
+    if AIIDA_LOGGER.level <= logging.INFO:
```

Comment:
should this not be `<= logging.REPORT` now

Reply:
(and same for other two instances below)

Reply:
fixed

Reply:
don't see the change yet?

Reply:
Should be there now. Were a bunch of other fixes that had to go in.